packages/lesswrong/lib/collections/digestPosts/schema.ts

```diff
@@ -0,0 +1,55 @@
+import { foreignKeyField } from '../../utils/schemaUtils'
+import { schemaDefaultValue } from '../../collectionUtils';
+
+export const DIGEST_STATUSES = ['yes', 'maybe', 'no'] as const
+export type InDigestStatuses = typeof DIGEST_STATUSES
+export type InDigestStatus = InDigestStatuses[number]
```

Comment:
We have this TupleSet pattern for this situation (there is a long comment by the definition explaining why it's better)
```suggestion
const DIGEST_STATUSES = new TupleSet(['yes', 'maybe', 'no'] as const);
export type ReviewSortOption = UnionOf<typeof DIGEST_STATUSES>;
```

Reply:
Oh nice - appreciate this, thanks!